server/src/controllers/galleryController.js

```diff
@@ -0,0 +1,67 @@
+import { Gallery } from "../models/gallerySchema.js";
+import { logger } from "../../logger.js";
+import multer from "multer";
+
+import { getAllImagesFromGallery, deleteImageFromGallery } from "../service/galleryService.js";
+
+const storage = multer.diskStorage({
+  destination: function (req, file, cb) {
+    cb(null, "../client/public/images"); 
```

Comment:
Your storage has a client path, what does client have to do with backend?

Reply:
Done